convert_to_hdf5/src/convert_to_hdf5.py

```diff
@@ -148,8 +149,46 @@ def get_topic_info(data_info_mapping, bag_file, meta_data):
         topic_info[vid_topic] = msg
     return topic_info
 
+def get_realsense_extrinsics(bag_file):
+    """Extracts extrinsics for all realsense cameras
 
-def load_hdf_files(record_names, out_dir, data_info_mapping, meta_data, topic_info):  # pylint: disable=too-many-locals
+    Args:
+        bag_file: the opened bag file
+
+    Returns: A dictionary with keys that are the names of the camera
+             ex: 'upper_realsense' and values that are a message of type
+             realsense2_camera/Extrinsics. These messages have three fields
+             `header`, `rotation`, `translation`. The rotation is 9 element
+             list, the translation is a 3 element list.
+    """
+    topics = bag_file.get_type_and_topic_info()[1].keys()
+    realsense_topics = filter(lambda val: 'realsense' in val, topics)
+    extrinsics_topics = filter(
+        lambda val: 'extrinsics' and 'depth_to_color' in val, realsense_topics)
```

Comment:
why both extrinsics and depth_to_color to get the extrinsics topics?

Reply:
I am not sure about this, I think I am using this code from you.

Reply:
lol

I see: 
![image](https://user-images.githubusercontent.com/5924806/118053962-74502900-b353-11eb-812e-e56e23ee4865.png)